docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+Creating a simple Menu
+======================
+
+### From open.mp Wiki
+
+
+
+Contents
+--------
+
+*   1 Menus in Pawn
+*   2 First menu steps
+*   3 Add some menu items
+*   4 Creating the effects for the items
+*   5 Last steps
+*   6 Last words
+*   7 A full test script for menus
+
+
+### 1 Menus in Pawn
+-------------
```

Comment:
You only need one heading style, the accepted form is `#` characters, so the `------` is unnecessary.